src/main/scala/com/wavesplatform/http/DebugApiRoute.scala

```diff
@@ -63,6 +64,8 @@ case class DebugApiRoute(ws: WavesSettings,
 
   import DebugApiRoute._
 
+  private[this] val rollbackScheduler = Scheduler.singleThread("debug-rollback")
```

Comment:
There's no need for a dedicated scheduler. `rollbackScheduler` is only used to perform some housekeeping, which can easily be done on one of API threads.